sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Watch.java

```diff
@@ -124,8 +124,7 @@
  * href="https://beam.apache.org/documentation/runners/capability-matrix/">capability matrix</a>.
  */
 @SuppressWarnings({
-  "nullness", // TODO(https://github.com/apache/beam/issues/20497)
-  "rawtypes"
+  "nullness" // TODO(https://github.com/apache/beam/issues/20497)
```

Comment:
the purpose of these change is try to flush bad gradle cache, which causes javacompile failing on these two files